plugins/cameraserver/src/main/java/edu/wpi/first/shuffleboard/plugin/cameraserver/data/CameraServerData.java

```diff
@@ -0,0 +1,50 @@
+package edu.wpi.first.shuffleboard.plugin.cameraserver.data;
+
+import com.google.common.collect.ImmutableMap;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import edu.wpi.first.shuffleboard.api.data.ComplexData;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import javafx.scene.image.Image;
+
+// serialversion UID doesn't matter since this is only used in one JVM at a time
+@SuppressFBWarnings("SE_NO_SERIALVERSIONID")
```

Comment:
Class is Serializable, but doesn't define serialVersionUID

>This class implements the Serializable interface, but does not define a serialVersionUID field.  A change as simple as adding a reference to a .class object will add synthetic fields to the class, which will unfortunately change the implicit serialVersionUID (e.g., adding a reference to String.class will generate a static field class$java$lang$String). Also, different source code to bytecode compilers may use different naming conventions for synthetic variables generated for references to class objects or inner classes. To ensure interoperability of Serializable across versions, consider adding an explicit serialVersionUID.

This seems like something reasonable to do.

Reply:
@SamCarlberg Are you arguing that because the object will never need to be serialized and read in by another JVM (even from another launch) this isn't necessary?